rust/agama-dbus-server/src/questions.rs

```diff
@@ -81,7 +83,12 @@ enum QuestionType {
 }
 
 /// Trait for objects that can provide answers to all kind of Question.
+///
+/// If not strategy is used or all defined strategies does not know answer,
```

Comment:
NP: a bit of rewording: "If no strategy is selected or the answer is unknown, then ask to the user".